<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>981</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab_AVL">
     <attribute name="title">
      <string>AVL</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView_AVL">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>941</width>
        <height>521</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_insertAVL">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_insertAVL">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Insert</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_deleteAVL">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_deleteAVL">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>11</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_delete_all_AVL">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>11</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Delete Tree</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_insertSomeAVL">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>11</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Insert N nodes</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_RB">
     <attribute name="title">
      <string> RB </string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView_RB">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>941</width>
        <height>521</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_insertRB">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Insert</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_insertRB">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_deleteRB">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>11</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_deleteRB">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_delete_all_RB">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>11</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Delete Tree</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_insertSomeRB">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>11</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Insert N nodes</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Treap">
     <attribute name="title">
      <string>Treap</string>
     </attribute>
     <widget class="QPushButton" name="button_insertTreap">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Insert</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_insertTreap">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_deleteTreap">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>11</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_deleteTreap">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_Treap">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>941</width>
        <height>521</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_delete_all_Treap">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>11</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Delete Tree</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_insertSomeTreap">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>11</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Insert N nodes</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Splay">
     <attribute name="title">
      <string>Splay</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit_insertSplay">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_insertSomeSplay">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>11</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Insert N nodes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_insertSplay">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Insert</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_deleteSplay">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_Splay">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>941</width>
        <height>521</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_deleteSplay">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>11</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_delete_all_Splay">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>11</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Delete Tree</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>630</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Medium</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>synchronize</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
